Sources/TokamakCore/Views/ForEach.swift

```diff
@@ -12,6 +12,19 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/// A structure that computes `View`s from a collection of identified data.
+///
+/// Available when `Data` conforms to `RandomAccessCollection`,
+/// `ID` conforms to `Hashable`, and `Content` conforms to `View`.
+///
+/// The children computed by `ForEach` are directly passed to the encapsulating `View`.
+/// Similar to `TupleView` and `Group`.
+///
+///     HStack {
+///       ForEach(0..<5) {
+///         Text("\($0)")
+///       }
+///     }
 public struct ForEach<Data, ID, Content>: View where Data: RandomAccessCollection, ID: Hashable, Content: View {
```

Comment:
<!--
  1 failure:  Line should be 10...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Sources/TokamakCore/Views/ForEach.swift;
  Line: 28;
-->
- :no_entry_sign: Line should be 100 characters or less: currently 112 characters (`line_length`)